postcss.config.js

```diff
@@ -1,23 +1,23 @@
 const path = require('path')
 
-const { production } = require('./config/webpack.vars')
+module.exports = ({ options }) => {
+  let plugins = [
+    require('postcss-import')({
+      path: [path.resolve(__dirname, 'src/styles/')]
+    }),
+    require('autoprefixer')(['last 2 versions', 'not dead', 'not ie <= 11'])
+  ]
 
-const commons = [
-  require('postcss-import')({
-    path: [path.resolve(__dirname, 'src/styles/')],
-    plugins: production ? [
-      require('autoprefixer')(['last 2 versions']),
```

Comment:
It seems that giving plugins to `postcss-import` doesn't work: the autoprefixer was adding the prefixes to the generated CSS.